packages/vue/src/components/organisms/SfTabs/_internal/SfTab.vue

```diff
@@ -1,18 +1,21 @@
 <template>
   <Fragment class="sf-tabs__tab">
-    <div
-      class="sf-tabs__title"
-      :class="{ 'sf-tabs__title--active': isActive }"
-      @click="tabClick"
-    >
-      {{ title }}
-      <div class="sf-tabs__chevron">
-        <SfChevron :class="{ 'sf-chevron--top': isActive }" />
+    <!--@slot Title. Here you should pass a title tab-->
+    <slot name="title" v-bind="{ tabClick }">
```

Comment:
```suggestion
    <slot name="title" v-bind="{ tabClick, isActive, title }">
```
@aniamusial because you use it inside slot :)